theseus_gui/src/store/instancesStore.js

```diff
@@ -0,0 +1,337 @@
+import { defineStore } from 'pinia'
+import { ofetch } from 'ofetch'
+import generated from '@/generated'
+
+export const useInstances = defineStore('instanceStore', {
+  state: () => ({
```

Comment:
It feels a bit messy to have the search stuff in here with the instances store. Could this logic be moved out to it's own store? ex: `searchStore.js`